{
  "uid" : "1d92c2efe0e96b26",
  "name" : "Verify presence, visibility, clickability, href, status code of links in sections",
  "fullName" : "tests.used_resources_page_test.TestUsedResourcesPageForAuthorizedUserLinks#test_ur_03_01_verify_links_in_sections",
  "historyId" : "e81f1089a6a1fbd12d91fa249d1a3eef",
  "time" : {
    "start" : 1728697371719,
    "stop" : 1728697371719,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5617c798a02a <unknown>\n#1 0x5617c76705e0 <unknown>\n#2 0x5617c76687eb <unknown>\n#3 0x5617c7658a09 <unknown>\n#4 0x5617c765a744 <unknown>\n#5 0x5617c7658cbd <unknown>\n#6 0x5617c765852f <unknown>\n#7 0x5617c7658422 <unknown>\n#8 0x5617c7656493 <unknown>\n#9 0x5617c7656ada <unknown>\n#10 0x5617c7672f41 <unknown>\n#11 0x5617c77048c5 <unknown>\n#12 0x5617c76e4b22 <unknown>\n#13 0x5617c7703d7d <unknown>\n#14 0x5617c76e48c3 <unknown>\n#15 0x5617c76b26b3 <unknown>\n#16 0x5617c76b368e <unknown>\n#17 0x5617c7954a2b <unknown>\n#18 0x5617c79589b1 <unknown>\n#19 0x5617c7941225 <unknown>\n#20 0x5617c7959532 <unknown>\n#21 0x5617c792638f <unknown>\n#22 0x5617c7978f28 <unknown>\n#23 0x5617c79790f3 <unknown>\n#24 0x5617c7988e7c <unknown>\n#25 0x7f7238494ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2a42a0e499a05b26660d8e26703607e5\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4df4ec9280>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5617c79790f3 \\\\u003Cunknown>\\\\n#24 0x5617c7988e7c \\\\u003Cunknown>\\\\n#25 0x7f7238494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5617c798a02a <unknown>\nE       #1 0x5617c76705e0 <unknown>\nE       #2 0x5617c76687eb <unknown>\nE       #3 0x5617c7658a09 <unknown>\nE       #4 0x5617c765a744 <unknown>\nE       #5 0x5617c7658cbd <unknown>\nE       #6 0x5617c765852f <unknown>\nE       #7 0x5617c7658422 <unknown>\nE       #8 0x5617c7656493 <unknown>\nE       #9 0x5617c7656ada <unknown>\nE       #10 0x5617c7672f41 <unknown>\nE       #11 0x5617c77048c5 <unknown>\nE       #12 0x5617c76e4b22 <unknown>\nE       #13 0x5617c7703d7d <unknown>\nE       #14 0x5617c76e48c3 <unknown>\nE       #15 0x5617c76b26b3 <unknown>\nE       #16 0x5617c76b368e <unknown>\nE       #17 0x5617c7954a2b <unknown>\nE       #18 0x5617c79589b1 <unknown>\nE       #19 0x5617c7941225 <unknown>\nE       #20 0x5617c7959532 <unknown>\nE       #21 0x5617c792638f <unknown>\nE       #22 0x5617c7978f28 <unknown>\nE       #23 0x5617c79790f3 <unknown>\nE       #24 0x5617c7988e7c <unknown>\nE       #25 0x7f7238494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728697371719,
      "stop" : 1728697372495,
      "duration" : 776
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728697167335,
      "stop" : 1728697167335,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728697372495,
      "stop" : 1728697372917,
      "duration" : 422
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5617c798a02a <unknown>\n#1 0x5617c76705e0 <unknown>\n#2 0x5617c76687eb <unknown>\n#3 0x5617c7658a09 <unknown>\n#4 0x5617c765a744 <unknown>\n#5 0x5617c7658cbd <unknown>\n#6 0x5617c765852f <unknown>\n#7 0x5617c7658422 <unknown>\n#8 0x5617c7656493 <unknown>\n#9 0x5617c7656ada <unknown>\n#10 0x5617c7672f41 <unknown>\n#11 0x5617c77048c5 <unknown>\n#12 0x5617c76e4b22 <unknown>\n#13 0x5617c7703d7d <unknown>\n#14 0x5617c76e48c3 <unknown>\n#15 0x5617c76b26b3 <unknown>\n#16 0x5617c76b368e <unknown>\n#17 0x5617c7954a2b <unknown>\n#18 0x5617c79589b1 <unknown>\n#19 0x5617c7941225 <unknown>\n#20 0x5617c7959532 <unknown>\n#21 0x5617c792638f <unknown>\n#22 0x5617c7978f28 <unknown>\n#23 0x5617c79790f3 <unknown>\n#24 0x5617c7988e7c <unknown>\n#25 0x7f7238494ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728697372496,
        "stop" : 1728697372917,
        "duration" : 421
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5617c798a02a <unknown>\n#1 0x5617c76705e0 <unknown>\n#2 0x5617c76687eb <unknown>\n#3 0x5617c7658a09 <unknown>\n#4 0x5617c765a744 <unknown>\n#5 0x5617c7658cbd <unknown>\n#6 0x5617c765852f <unknown>\n#7 0x5617c7658422 <unknown>\n#8 0x5617c7656493 <unknown>\n#9 0x5617c7656ada <unknown>\n#10 0x5617c7672f41 <unknown>\n#11 0x5617c77048c5 <unknown>\n#12 0x5617c76e4b22 <unknown>\n#13 0x5617c7703d7d <unknown>\n#14 0x5617c76e48c3 <unknown>\n#15 0x5617c76b26b3 <unknown>\n#16 0x5617c76b368e <unknown>\n#17 0x5617c7954a2b <unknown>\n#18 0x5617c79589b1 <unknown>\n#19 0x5617c7941225 <unknown>\n#20 0x5617c7959532 <unknown>\n#21 0x5617c792638f <unknown>\n#22 0x5617c7978f28 <unknown>\n#23 0x5617c79790f3 <unknown>\n#24 0x5617c7988e7c <unknown>\n#25 0x7f7238494ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"2a42a0e499a05b26660d8e26703607e5\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728697372949,
      "stop" : 1728697373011,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Used Resources Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "used_resources_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestUsedResourcesPage"
  }, {
    "name" : "host",
    "value" : "fv-az570-490"
  }, {
    "name" : "thread",
    "value" : "2304-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.used_resources_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 30,
        "unknown" : 0,
        "total" : 33
      },
      "items" : [ {
        "uid" : "4759e099a89fa68f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/4759e099a89fa68f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x564ca123b02a <unknown>\n#1 0x564ca0f215e0 <unknown>\n#2 0x564ca0f197eb <unknown>\n#3 0x564ca0f09a09 <unknown>\n#4 0x564ca0f0b744 <unknown>\n#5 0x564ca0f09cbd <unknown>\n#6 0x564ca0f0952f <unknown>\n#7 0x564ca0f09422 <unknown>\n#8 0x564ca0f07493 <unknown>\n#9 0x564ca0f07ada <unknown>\n#10 0x564ca0f23f41 <unknown>\n#11 0x564ca0fb58c5 <unknown>\n#12 0x564ca0f95b22 <unknown>\n#13 0x564ca0fb4d7d <unknown>\n#14 0x564ca0f958c3 <unknown>\n#15 0x564ca0f636b3 <unknown>\n#16 0x564ca0f6468e <unknown>\n#17 0x564ca1205a2b <unknown>\n#18 0x564ca12099b1 <unknown>\n#19 0x564ca11f2225 <unknown>\n#20 0x564ca120a532 <unknown>\n#21 0x564ca11d738f <unknown>\n#22 0x564ca1229f28 <unknown>\n#23 0x564ca122a0f3 <unknown>\n#24 0x564ca1239e7c <unknown>\n#25 0x7f7992294ac3 <unknown>",
        "time" : {
          "start" : 1728611062944,
          "stop" : 1728611062944,
          "duration" : 0
        }
      }, {
        "uid" : "54fdabc34de79f43",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/54fdabc34de79f43",
        "status" : "passed",
        "time" : {
          "start" : 1728525377381,
          "stop" : 1728525380814,
          "duration" : 3433
        }
      }, {
        "uid" : "6b217205dca88839",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/6b217205dca88839",
        "status" : "passed",
        "time" : {
          "start" : 1728439207152,
          "stop" : 1728439218533,
          "duration" : 11381
        }
      }, {
        "uid" : "40997ed30e11f00",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/40997ed30e11f00",
        "status" : "passed",
        "time" : {
          "start" : 1728352651376,
          "stop" : 1728352654690,
          "duration" : 3314
        }
      }, {
        "uid" : "ee0b111d6a786f9d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/ee0b111d6a786f9d",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728267811356,
          "stop" : 1728267811356,
          "duration" : 0
        }
      }, {
        "uid" : "e11d5e60610b317b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/e11d5e60610b317b",
        "status" : "passed",
        "time" : {
          "start" : 1728180186609,
          "stop" : 1728180188782,
          "duration" : 2173
        }
      }, {
        "uid" : "f6194f6d18f087f1",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/f6194f6d18f087f1",
        "status" : "passed",
        "time" : {
          "start" : 1728093487082,
          "stop" : 1728093489129,
          "duration" : 2047
        }
      }, {
        "uid" : "58f0f0d2bd14e2ae",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/58f0f0d2bd14e2ae",
        "status" : "passed",
        "time" : {
          "start" : 1728007192662,
          "stop" : 1728007194747,
          "duration" : 2085
        }
      }, {
        "uid" : "a693f187384c7ce3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/a693f187384c7ce3",
        "status" : "passed",
        "time" : {
          "start" : 1727921058958,
          "stop" : 1727921060730,
          "duration" : 1772
        }
      }, {
        "uid" : "65b1432cb09547b8",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/65b1432cb09547b8",
        "status" : "passed",
        "time" : {
          "start" : 1727834360183,
          "stop" : 1727834364879,
          "duration" : 4696
        }
      }, {
        "uid" : "8b4c07cb81556b83",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/8b4c07cb81556b83",
        "status" : "passed",
        "time" : {
          "start" : 1727748515404,
          "stop" : 1727748519176,
          "duration" : 3772
        }
      }, {
        "uid" : "36e07e1cdf13b0b7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/36e07e1cdf13b0b7",
        "status" : "passed",
        "time" : {
          "start" : 1727661768721,
          "stop" : 1727661770918,
          "duration" : 2197
        }
      }, {
        "uid" : "10eeb396e673e416",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/10eeb396e673e416",
        "status" : "passed",
        "time" : {
          "start" : 1727575689128,
          "stop" : 1727575700999,
          "duration" : 11871
        }
      }, {
        "uid" : "1ca3132c810c3d55",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/1ca3132c810c3d55",
        "status" : "passed",
        "time" : {
          "start" : 1727489023313,
          "stop" : 1727489026119,
          "duration" : 2806
        }
      }, {
        "uid" : "93945595740eccd8",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/93945595740eccd8",
        "status" : "passed",
        "time" : {
          "start" : 1727402727408,
          "stop" : 1727402730241,
          "duration" : 2833
        }
      }, {
        "uid" : "fc47a8c501689f0a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/fc47a8c501689f0a",
        "status" : "passed",
        "time" : {
          "start" : 1727316282454,
          "stop" : 1727316288337,
          "duration" : 5883
        }
      }, {
        "uid" : "66a47bc51a41157",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/66a47bc51a41157",
        "status" : "passed",
        "time" : {
          "start" : 1727229889614,
          "stop" : 1727229891730,
          "duration" : 2116
        }
      }, {
        "uid" : "eb8394803cfabb2f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/eb8394803cfabb2f",
        "status" : "passed",
        "time" : {
          "start" : 1727143214894,
          "stop" : 1727143217137,
          "duration" : 2243
        }
      }, {
        "uid" : "7b2ecaaf35599f8d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/7b2ecaaf35599f8d",
        "status" : "passed",
        "time" : {
          "start" : 1727056646987,
          "stop" : 1727056649118,
          "duration" : 2131
        }
      }, {
        "uid" : "ad92919057603596",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/40//#testresult/ad92919057603596",
        "status" : "passed",
        "time" : {
          "start" : 1726970470762,
          "stop" : 1726970478822,
          "duration" : 8060
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1d92c2efe0e96b26.json",
  "parameterValues" : [ ]
}